smriprep/workflows/anatomical.py

```diff
@@ -685,7 +686,7 @@ def _split_segments(in_file):
     hdr.set_data_dtype('uint8')
 
     out_files = []
-    for i, label in enumerate(("CSF", "WM", "GM"), 1):
+    for i, label in enumerate(("GM", "WM", "CSF"), 1):
```

Comment:
this matches up with bids derivatives 👍 

does `segimg` match up to these labels? if so, we were producing the wrong inputs for some time?

Reply:
Looks like so, which puzzles me a bit because I checked several times on this.

Reply:
Please note that this was an incidental finding - I was checking the correctness on both ``--fs-no-reconall`` and with reconall paths and discovered this.

There are two more combinations to this, corresponding to the equivalent paths using the ``--anat-derivatives`` flag.

Reply:
This has been out with rc4 and rc5. Have we released fMRIPrep 20.0.x depending on any of these RCs?

Reply:
Okay, nope (https://github.com/poldracklab/fmriprep/blob/20.0.7/setup.cfg). Which is great news - thanks @effigies for the version dependency matrix - that saved our asses this time :)